core/iwasm/common/gc/gc_type.c

```diff
@@ -745,13 +750,26 @@ wasm_is_reftype_supers_of_extern(uint8 type)
     return (type == REF_TYPE_EXTERNREF) ? true : false;
 }
 
+#if WASM_ENABLE_STRINGREF != 0
+inline static bool
+wasm_is_reftype_supers_of_string(uint8 type)
+{
+    return (type == REF_TYPE_STRINGREF || type == REF_TYPE_ANYREF) ? true
+                                                                   : false;
+}
+#endif
+
 inline static bool
 wasm_is_reftype_supers_of_none(uint8 type, const WASMRefType *ref_type,
                                const WASMTypePtr *types, uint32 type_count)
 {
     if (type == REF_TYPE_NULLREF || type == REF_TYPE_I31REF
         || type == REF_TYPE_STRUCTREF || type == REF_TYPE_ARRAYREF
-        || wasm_is_reftype_supers_of_eq(type))
+        || wasm_is_reftype_supers_of_eq(type)
+#if WASM_ENABLE_STRINGREF != 0
+        || type == REF_TYPE_STRINGREF
```

Comment:
Can type be REF_TYPE_STRINGVIEWWTF8/REF_TYPE_STRINGVIEWWTF16/REF_TYPE_STRINGVIEWITER?

Reply:
No, these stringview types are not in the subtyping chain, they are created from stringref when really needed.